doc/_ext/collapsible/collapsible.py

```diff
@@ -45,7 +45,9 @@ def run(self):
         collapsible_id_counter += 1
         self.state.nested_parse(self.content, self.content_offset, collapsible_node)
 
-        self.state.document.settings.env.config.html_static_path.append(os.path.dirname(__file__))
+        directory = os.path.dirname(__file__)
+        if not directory in self.state.document.settings.env.config.html_static_path:
+            self.state.document.settings.env.config.html_static_path.append(directory)
```

Comment:
The path must start one level above the extension directory. This is needed to include the extension name when adding css.

Reply:
Unfortunately, the css file has to be placed into separate directory.
With current approach, extension source file is also published into html directory.